Add post-channel creation steps to Slack page

ddbeck opened the pull request:
Previously undocumented, this PR explains what happens _after_ we create a channel in Slack.

<!-- readthedocs-preview writethedocs-www start -->
----
:books: Documentation preview :books:: https://writethedocs-www--1995.org.readthedocs.build/

<!-- readthedocs-preview writethedocs-www end -->

ddbeck committed:
Add post-channel creation steps to Slack page

ddbeck committed:
Less channel channel channel channel

plaindocs commented:
Might be worth supplying an actual example of an announcement, either real or artificial?

janine-c reviewed (approved):
:shipit:

ddbeck committed:
Add example channel announcement

ddbeck commented:
> Might be worth supplying an actual example of an announcement, either real or artificial?

Good idea. I added a joke one, though we probably ought to replace it with something real (after the next time it happens, I guess).

ravindk89 commented on docs/slack.rst:
```diff
@@ -168,6 +168,12 @@ If you still can't find a suitable channel, you can propose a new channel. To pr
 The Write the Docs moderation team will review your proposal.
+If your proposal is approved, then you and your consponsors must:
+
+#. Seed discussion in the new channel.
```
```suggestion
#. Make a best-effort to seed and maintain a cadence of conversation.
    For example, you can post questions once a week related to the channel topic and encourage others to participate, similar to how **#learn-tech-writing** contributors have done book clubs.
```

ravindk89 replied:
I'd like to set the bar to be "If you're going to start this up, you need to do more than show up at the beginning"

ddbeck replied:
We've had some questions in the past from folks who were worried they were taking on a permanent commitment by suggesting a channel (e.g., "do I have to moderate the channel, if I propose it?"). I think putting in a requirement like this might scare people off more than it would spur conversation.

ravindk89 commented on docs/slack.rst:
```diff
@@ -168,6 +168,12 @@ If you still can't find a suitable channel, you can propose a new channel. To pr
+   For example, ask an open-ended question, share a link to discuss, or post a hot take.
+#. Make an announcement in **#general** and channels related to the new channel's topic.
+   For example, if there's a new a channel for underwater basket weaving, then make an announcement like "Join us in #underwater-basket-weaving to discuss techniques, materials, and what to store in all these baskets."
 
```
Do we add down here something to the effect of "Channels which remain largely empty or quiet may be closed by the Moderation team."

ddbeck replied:
I don't think we should make a threat we're probably never going to act on. I've never got myself together enough to do some relatively easy channel renaming. I just don't see the mod team actively _looking_ for empty channels to kill.

ravindk89 commented:
All nits, nothing to block an existing LGTM

ddbeck reviewed (commented):
Janine had some ideas for me in chat, so I'm going to put this into draft mode and do some rework.

ddbeck committed:
Explain why we use this channel creation method

ddbeck commented:
OK, I've redone this a little emphasize _why_ we're doing this: to help conversations happen (and to head off the idea that it's a gatekeeping exercise). I've also asked channel proposers to think of their seed conversation earlier in the process, so it's not a surprise and they're likelier to remember to do it, even if we forget to ask.

ddbeck committed:
Add Slack channel proposal review reasoning

@janine-c:

https://github.com/writethedocs/www/commit/be680b0c5f8eac050b9ab8b53c4259c19674abec#r122875538

ddbeck commented on docs/slack.rst:
```diff
@@ -161,13 +161,26 @@ Creating channels
+#. Start thinking about how to seed the new channel with discussion, with an open-ended question, a link to discuss, or a hot take.
+
+The Write the Docs moderation team will review your proposal to make sure the new channel will be active, durable, and won't duplicate or create confusion with another channel.
+Whenever possible, the moderators work with channel sponsors on improvements to the proposal.
```
@janine-c what do you think of the revisions here? Following up on https://github.com/writethedocs/www/commit/be680b0c5f8eac050b9ab8b53c4259c19674abec#r122875538

janine-c replied:
Love it! I wonder if "durable" will confuse people - maybe "sustainable" might make more sense? Are we thinking about making sure that the channel will stay active/useful over time here?

ddbeck replied:
Yes, I could see that might be confusing. I was trying to capture the idea of "not temporary" (e.g., I don't want people suggesting channels that are for one-off events). A couple of alternatives might be:

- Long-lived
- Permanent
- Enduring

Or I could just omit it and go with "…active and won't duplicate…".

Any thoughts?

janine-c replied:
Yeah, maybe we could say something like "...the new channel will remain active and won't duplicate..."? I think that even when channels feel like they'll probably meet that criteria, we can't know if they'll be permanent, but I think that phrasing hits on the idea that we don't want to have a channel that's, say, only related to a single year's hackathon or anything like that.

ddbeck replied:
"Remain active" I like it, done! ✅

ddbeck commented on docs/slack.rst:
```diff
@@ -161,13 +161,26 @@ Creating channels
+   Be sure to share your proposed channel name, description, and to tag your cosponsors.
+#. Start thinking about how to seed the new channel with discussion, with an open-ended question, a link to discuss, or a hot take.
+
+The Write the Docs moderation team will review your proposal to make sure the new channel will be active, durable, and won't duplicate or create confusion with another channel.
```
Suggestion from https://github.com/writethedocs/www/pull/1995/files#r1306712473

```suggestion
The Write the Docs moderation team will review your proposal to make sure the new channel will remain active and won't duplicate or create confusion with another channel.
```

ddbeck committed:
Fix confusing "durable" channel wording

ddbeck committed:
Merge branch 'main' into channel-creation-update

ddbeck committed:
Merge branch 'main' into channel-creation-update
